Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM node:alpine
+FROM node:19.0.0-alpine
 WORKDIR /src/
 COPY ./themes/conventional-commits /src/
 RUN apk add python make g++
```

Comment:
i think that python and g++ are not required anymore now that we have removed node-sass dependency